libqtile/scripts/start.py

```diff
@@ -89,7 +90,7 @@ def start(options):
     rename_process()
     q = make_qtile(options)
     try:
-        q.loop()
+        asyncio.run(q.loop())
```

Comment:
Can we make the API simpler and keep `loop` as a synchronous function that calls `asyncio.run` itself, as `SessionManager` used to?